### DISCUSSION TITLE
[QUESTION]How to show tour guide only once for the user?

### DISCUSSION THREAD

**Discussion by vegeta123254** [elrumordelaluz/reactour#398] [Q&A] [answered]
I have used tour guide for a new feature onboarding, how do I show the tour guide once for the user? 
I have setSteps in each component. 

Anything I can do with `TourProvider` ?

**Comment by elrumordelaluz** [comment, marked as answer]
Hi @vegeta123254, should be easier if you can create a minimal reproduction in a sandbox. However, the way you show the _Tour_ to the user is up to you, ie. open once the user lands in a page, click a button and so on. So to show once, depends on how you are showing/triggering the _Tour_.
